integration_tests/dbt_project.yml

```diff
@@ -27,6 +27,10 @@ vars:
     servicenow_sys_user_has_role_identifier: "sys_user_has_role_data"
     servicenow_sys_user_grmember_identifier: "sys_user_grmember_data"
 
+    servicenow__using_roles: true # to be turned on when generating docs so that can be generated via the manifest and docs as it is turned off by default.
+
+# models:
+#   +schema: "servicenow_{{ var('directed_schema','dev') }}"
```

Comment:
We can keep this uncommented. It won't impact the integration test runs.

Reply:
sure thing!